ui/src/sandbox.rs

```diff
@@ -190,19 +190,23 @@ impl Sandbox {
 
         let mut cmd = Command::new("docker");
 
-        cmd
-            .arg("run")
+        cmd.arg("run")
             .arg("--rm")
-            .arg("--volume").arg(&mount_input_file)
-            .arg("--volume").arg(&mount_output_dir)
+            .arg("--volume")
+            .arg(&mount_input_file)
+            .arg("--volume")
+            .arg(&mount_output_dir)
```

Comment:
Could you undo these changes? I'm attempting to keep it as `name, value` to more easily match up what it does.